requirements_test.txt

```diff
@@ -1,47 +1,43 @@
-# linters such as flake8 and pylint should be pinned, as new releases
-# make new things fail. Manually update these pins when pulling in a
-# new version
-
-# types-* that have versions roughly corresponding to the packages they
-# contain hints for available should be kept in sync with them
-
+# Automatically generated by gen_requirements_all.py, do not edit
 -c homeassistant/package_constraints.txt
 -r requirements_test_pre_commit.txt
 astroid==2.12.12
 codecov==2.1.12
 coverage==6.4.4
+fnvhash==0.1.0
```

Comment:
This is not a test dependency, but an integration dependency. It doesn't belong here.

Reply:
It is currently required to run any test due to `recorder`'s early import in `conftest`. Please see the PR description/pyproject.toml comment discussion.

Reply:
Unresolving conversation as it is not resolved.